notify/notify.go

```diff
@@ -309,7 +321,7 @@ func (pb *PipelineBuilder) New(
 	wait func() time.Duration,
 	inhibitor *inhibit.Inhibitor,
 	silencer *silence.Silencer,
-	muteTimes map[string][]timeinterval.TimeInterval,
+	times map[string][]timeinterval.TimeInterval,
```

Comment:
IIUC we're still passing only the mute time intervals?

Reply:
sorry what means IIUC?

Reply:
My bad, IIUC="if I understand correctly".

I meant to say that the main() function only passes `conf.MuteTimeIntervals` and that it should merge `MuteTimeIntervals` and `TimeIntervals` instead.

https://github.com/prometheus/alertmanager/blob/da9de1f540af4f01f6336d4424e76a70c547bcbb/cmd/alertmanager/main.go#L439-L443

Reply:
Yes, that was right I update that part to merge the both configs